bundles/org.openhab.binding.keba/src/main/resources/ESH-INF/thing/kecontact.xml

```diff
@@ -218,16 +218,16 @@
 		<state readOnly="false"></state>
 	</channel-type>
 	<channel-type id="authreq" advanced="true">
-		<item-type>Number</item-type>
+		<item-type>Switch</item-type>
 		<label>Authentication Required</label>
 		<description>Authentication required</description>
-		<state readOnly="true"></state>
+		<state pattern="%d" readOnly="true"></state>
```

Comment:
Shouldn't you also revert this change? `%d` isn't a suitable pattern for a Switch.